x-pack/test/functional_with_es_ssl/apps/uptime/anomaly_alert.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  describe('uptime anomaly alert', () => {
+    const pageObjects = getPageObjects(['common', 'uptime']);
+    const supertest = getService('supertest');
+    const retry = getService('retry');
+
+    const monitorId = '0000-intermittent';
+
+    const uptime = getService('uptime');
+
+    const DEFAULT_DATE_START = 'Sep 10, 2019 @ 12:40:08.078';
+    const DEFAULT_DATE_END = 'Sep 11, 2019 @ 19:40:08.078';
+    let alerts: any;
+    const alertId = 'uptime-anomaly-alert';
+
+    before(async () => {
+      alerts = getService('uptime').alerts;
+
+      await uptime.navigation.goToUptime();
+
+      await uptime.navigation.loadDataAndGoToMonitorPage(
+        DEFAULT_DATE_START,
+        DEFAULT_DATE_END,
+        monitorId
+      );
+    });
+
+    it('can delete existing job', async () => {
+      if (await uptime.ml.alreadyHasJob()) {
+        await uptime.ml.openMLManageMenu();
+        await uptime.ml.deleteMLJob();
+        await uptime.navigation.refreshApp();
+      }
+    });
+
+    it('can open ml flyout', async () => {
+      await uptime.ml.openMLFlyout();
+    });
+
+    it('has permission to  create job', async () => {
+      expect(uptime.ml.canCreateJob()).to.eql(true);
+      expect(uptime.ml.hasNoLicenseInfo()).to.eql(false);
+    });
+
+    it('can create job successfully', async () => {
+      await uptime.ml.createMLJob();
```

Comment:
It looks like this tests both anomaly alerts and enabling ML. We should just move the existing ML tests here to prevent duplication after FF .